packages/block-editor/src/components/block-navigation/index.js

```diff
@@ -40,8 +40,15 @@ function BlockNavigationList( {
 				 * document.getElementsByClassName("editor-block-navigation__container")[0].closest(".components-popover__content").scrollTop
 				 *		= document.getElementsByClassName("components-button is-selected")[0].offsetTop
 				 */
+
 				const blockEl = useRef( null );
-				// selected_block.current.closest(".components-popover__content").scrollTop = block.offsetTop;
+
+				// similar to componentDidMount and componentDidUpdate
+				useEffect( () => {
+					if ( isSelected ) {
+						blockEl.current.closest(".components-popover__content").scrollTop = block.offsetTop;
```

Comment:
I read into `useEffect(...)` and this seems like the way forward, but my `blockEl` doesn't seem to have the `closest(...)` property but the other `Element` properties [here](https://developer.mozilla.org/en-US/docs/Web/API/Element). Am i missing something?
![gb14311-closest-not-available](https://user-images.githubusercontent.com/12999836/56232365-95e46680-604e-11e9-818b-70d57aee730a.png)